styleguide/components/21-atoms/heading/css/_overview-title.scss

```diff
@@ -17,15 +16,11 @@ h6 {
       font-size: inherit;
 
       &::after {
-        display: block;
-        position: absolute;
-        right: 0;
-        bottom: -.1em;
-        left: 0;
-        width: 70%;
-        margin: auto;
-        border-bottom: 2px solid;
-        content: '';
+        margin-left: .5rem;
+        font-size: 3.5rem;
+        line-height: .5;
+        content: ':';
```

Comment:
Is de : wel degelijk gelijk aan de font? Ik begreep steeds dat het een img is. Lezen de screen readers dit mogelijks ook niet voor?

Reply:
Het is wel degelijk de font blijkbaar.

Reply:
Mogelijks nog een issue met screen readers?

Reply:
Fix toegevoegd.